src/common/conf/conf.cpp

```diff
@@ -261,6 +261,7 @@ expected::ExpectedSize MenderConfig::ProcessCmdlineArgs(
 	http_client_config_.client_cert_path = https_client.certificate;
 	http_client_config_.client_cert_key_path = https_client.key;
 	http_client_config_.skip_verify = skip_verify;
+	http_client_config_.disable_keep_alive = connectivity.disable_keep_alive;
```

Comment:
Argh! I was so meticulous when I went through the config. Did I really miss it?? :sob:

Reply:
This is one out of many, your catch ratio was still very high!

Reply:
There was also a `ssl_engine` setting missing. Just to throw some more coal on the fire :p

Reply:
Haha, you couldn't pass up that chance, could you! 😃

Reply:
hahaha, no :laughing: